adapters/src/main/kotlin/com/krzykrucz/elesson/currentlesson/startlesson/FetchClassRegistryAdapter.kt

```diff
@@ -0,0 +1,41 @@
+package com.krzykrucz.elesson.currentlesson.startlesson
+
+import arrow.core.Either
+import com.krzykrucz.elesson.currentlesson.domain.AsyncOutput
+import com.krzykrucz.elesson.currentlesson.domain.NaturalNumber
+import com.krzykrucz.elesson.currentlesson.domain.NonEmptyText
+import com.krzykrucz.elesson.currentlesson.domain.startlesson.ClassName
+import com.krzykrucz.elesson.currentlesson.domain.startlesson.ClassRegistry
+import com.krzykrucz.elesson.currentlesson.domain.startlesson.FetchClassRegistry
+import com.krzykrucz.elesson.currentlesson.domain.startlesson.FirstName
+import com.krzykrucz.elesson.currentlesson.domain.startlesson.NumberInRegister
+import com.krzykrucz.elesson.currentlesson.domain.startlesson.SecondName
+import com.krzykrucz.elesson.currentlesson.domain.startlesson.StudentRecord
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+
+class ERegisterClient : FetchClassRegistry {
+    override fun invoke(p1: ClassName): AsyncOutput<ClassRegistry, Throwable> =
+        listOf(
+            StudentRecord(
+                FirstName(NonEmptyText.of("Harry")!!),
+                SecondName(NonEmptyText.of("Potter")!!),
+                NumberInRegister(NaturalNumber.ONE)
+            ),
+            StudentRecord(
+                FirstName(NonEmptyText.of("Hermione")!!),
+                SecondName(NonEmptyText.of("Granger")!!),
+                NumberInRegister(NaturalNumber.TWO)
+            )
+        )
+            .let { ClassRegistry(it, p1) }
+            .let { AsyncOutput.just(Either.right(it)) }
+
+}
+
+@Configuration
+class FetchClassRegistryAdapterConfig {
+    @Bean
+    fun fetchClassRegistryAdapter(): FetchClassRegistry = ERegisterClient()
```

Comment:
Do we need that as a bean? I don't think that it should be injected anywhere besides `createStartLesson` function. I'd just call a constructor in DomainConfig to only have a one bean per domain context.